src/ims.js

```diff
@@ -372,7 +372,7 @@ class Ims {
    *      {@link toTokenResult} or rejects to an error message.
    */
   async getAccessTokenByClientCredentials (clientId, clientSecret, orgId, scopes) {
-    aioLogger.debug('getAccessTokenServerToServer(%s, %s, %s, %o)', clientId, clientSecret, orgId, scopes = [])
+    aioLogger.debug('getAccessTokenByClientCredentials(%s, %s, %s, %o)', clientId, clientSecret, orgId, scopes = [])
```

Comment:
please don’t log the clientSecret, this is sensitive info like a password

Reply:
Thanks -- I'll truncate it to the last n chars.

Reply:
actually this should be ok since we are logging in a debug context only. However it seems that this type of logging is not isolated to just this function, but quite a few functions as well (as well as logging of the tokens, not just secrets). I'll bring this up for dev discussion as a separate issue for resolution: https://github.com/adobe/aio-lib-ims/issues/121